### PR TITLE
Add tags to paths in ocpaths.proto.

### PR DESCRIPTION
**Pull request by robshakir**
This change allows paths to be tagged with arbitrary criteria that allow for grouping or other identification of a path within a list.

### CODE REVIEW

File: proto/ocpaths.proto [hunk 1]
```diff
@@ -52,6 +54,10 @@ message OCPath {
   // This destination featureprofiles folder serves as a grouping of the paths
   // related to the tests which appear in this folder and subfolders the folder.
   string featureprofileid = 3;
+
+  // A set of opaque tags that are used for this path. These tags can be used
+  // to group paths according to use-case specific criteria.
+  repeated string paths = 4;
```
Comment (wenovus) [inline proto/ocpaths.proto:60]: paths -> tags?

Comment (robshakir) [reply proto/ocpaths.proto]: Good catch, fixed. Thanks.

**Comment by robshakir** [comment]
We should skip the `proto` directory for static analysis checks in this repo, I think.

**Review by wenovus — APPROVED** [review]
LGTM, @dplore you want to take a final look?

**Review by dplore — APPROVED** [review]

**Pull request merged**